README.md

```diff
@@ -56,6 +56,7 @@ set :composer_working_dir, -> { fetch(:release_path) }
 set :composer_dump_autoload_flags, '--optimize'
 set :composer_download_url, "https://getcomposer.org/installer"
 set :composer_version, '1.0.0-alpha8' #(default: not set)
+set :composer_php_version, :php72 #(default: :php) Or use full path /usr/bin/php72
```

Comment:
I wonder whether this might be clearer as `composer_php_executable`, since aren't really defining the version as much as the path. What do you think?

```suggestion
set :composer_php_executable, :php72 #(default: :php) Or use full path "/usr/bin/php72"
```